vite.config.ts

```diff
@@ -7,10 +7,10 @@ export default defineConfig(() => {
     plugins: [react()],
     test: {
       coverage: {
-        provider: "istanbul",
+        provider: "istanbul"
       },
       globals: true,
-      environment: "jsdom",
-    },
+      environment: "happy-dom"
```

Comment:
would you care to explain why?

Reply:
In one of the articles, they were suggesting to use it. Maybe I should start adding comments for situations when I change configuration to solve some problem and it's based on the different information found in web? In another branch I reverted it to  `environment: "jsdom"`